src/components/KillButton.js

```diff
@@ -1,119 +1,142 @@
-import React, { useState } from 'react';
-import { Flex, Button, Select, Box, Alert, AlertIcon, AlertTitle, AlertDescription } from '@chakra-ui/react';
+import React, { useEffect, useState } from 'react';
 import { db } from '../utils/firebase';
-import { query, updateDoc, collection, getDocs, where } from "firebase/firestore";
-import RemapPlayers from './RemapPlayers';
+import { collection, query, getDocs, where, onSnapshot, updateDoc } from "firebase/firestore";
+import { Select, Flex, Alert, AlertIcon, AlertTitle, AlertDescription, Box, HStack } from '@chakra-ui/react';
+import killImage from '../assets/kill.png'; // Adjust the path according to your project structure
 
 const KillButton = (props) => {
-    const [selectedPlayer, setSelectedPlayer] = useState('');
-    const [showAlert, setShowAlert] = useState(false); // State to control the alert visibility
+    const { roomID, assassinPlayerNamed, handleKillPlayer } = props;
```

Comment:
For consistency, can we move the props into the initialization of the Component? So: Kill Button = ({ roomID, assassinPlayerNamed, handleKillPlayer })